includes/gateways/paypal/admin/connect.php

```diff
@@ -252,122 +254,132 @@ function get_account_info() {
 		wp_send_json_error( wpautop( __( 'You do not have permission to perform this action.', 'easy-digital-downloads' ) ) );
 	}
 
-	$mode = edd_is_test_mode() ? PayPal\API::MODE_SANDBOX : PayPal\API::MODE_LIVE;
-
 	try {
-		/**
-		 * First get account information.
-		 */
-		$api      = new PayPal\API();
-		$response = $api->make_request( 'v1/identity/oauth2/userinfo?schema=paypalv1.1', array(), array(), 'GET' );
-
-		if ( empty( $response->user_id ) ) {
-			wp_send_json_error( wpautop( __( 'Unable to retrieve account information from PayPal. You may wish to try reconnecting.', 'easy-digital-downloads' ) ) );
-		}
+		$status         = 'success';
+		$account_status = '';
+		$actions = array(
+			'refresh_merchant' => '<button type="button" class="button edd-paypal-connect-action" data-nonce="' . esc_attr( wp_create_nonce( 'edd_check_merchant_status' ) ) . '" data-action="edd_paypal_commerce_check_merchant_status">' . esc_html__( 'Re-Check Payment Status', 'easy-digital-downloads' ) . '</button>',
+			'webhook'          => '<button type="button" class="button edd-paypal-connect-action" data-nonce="' . esc_attr( wp_create_nonce( 'edd_update_paypal_webhook' ) ) . '" data-action="edd_paypal_commerce_update_webhook">' . esc_html__( 'Sync Webhook', 'easy-digital-downloads' ) . '</button>'
```

Comment:
I clicked the "Sync Webhook" button just because it was there and I could and got this fun output:

![image](https://user-images.githubusercontent.com/4853066/123644759-ad553780-d7f3-11eb-983b-7807752a360b.png)

> Invalid response code 400 while syncing webhook. Response: {"name":"WEBHOOK_PATCH_REQUEST_NO_CHANGE","message":"No change in webhook","debug_id":"7213869913d9a","information_link":"https:\/\/developer.paypal.com\/docs\/api\/webhooks\/#errors","details":[],"links":[]}

Reply:
Huh. It sounds like you're getting that error because there are no changes to make. (As in, your webhook is already just fine.) However, I can't reproduce this. O_O

Reply:
Pushed up a change that might theoretically fix it https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8723/commits/9d97c1930cac529b1cb859255a0370fde1580c6c

Reply:
Yes, that fixed it!